ui/components/component-library/text/README.mdx

```diff
@@ -166,6 +166,33 @@ import { FontStyle } from '../../../helpers/constants/design-system';
 </Text>
 ```
 
+### Font Family
+
+Use the `fontFamily` prop and the `FontFamily` enum from `./text.types` to change the font family of the `Text` component. There are 3 font families:
+
+- `FontFamily.Default` (CentraNo1)
+- `FontFamily.Accent` (MMSans)
+- `FontFamily.Hero` (MMPoly)
+
+<Canvas>
+  <Story id="components-componentlibrary-text--font-family-story" />
+</Canvas>
+
+```jsx
+import { Text } from '../../component-library';
+import { FontFamily } from '../../../helpers/constants/design-system';
+
+<Text fontFamily={FontFamily.Default}>
+  Default Font (CentraNo1)
+</Text>
+<Text fontFamily={FontFamily.Accent}>
+  Accent Font (MMSans)
+</Text>
+<Text fontFamily={FontFamily.Hero}>
+  Hero Font (MMPoly)
+</Text>
+```
```

Comment:
Adds documentation in storybook

<img width="1511" alt="Screenshot 2025-05-13 at 1 18 57 PM" src="https://github.com/user-attachments/assets/9ab8c1c4-36d8-42b8-9fea-03927635c3ec" />